apps/accounts/verify.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import requests
+
+log = logging.getLogger('accounts.verify')
+ProfileNotFound = LookupError
+
+
+def fxa_identify(code, config=None):
+    """Get an FxA profile for an access token. If identification fails return
+    an empty dict."""
+    token = get_fxa_token(code, config)['access_token']
+    return get_fxa_profile(token, config)
+
+
+def get_fxa_token(code, config):
+    log.debug('Getting token for {code}'.format(code=code))
+    response = requests.post(config['oauth_uri'] + '/token', data={
+        'code': code,
+        'client_id': config['client_id'],
+        'client_secret': config['client_secret'],
+    })
+    if response.status_code == 200:
+        data = response.json()
+        if data.get('access_token'):
+            log.debug('Got token {data}'.format(data=data))
+            return data
+        else:
+            log.info('No token returned {data}'.format(data=data))
```

Comment:
it would be nice to have `{code}` in each of these log messages for traceability